indico/queries/model_groups.py

```diff
@@ -110,14 +110,16 @@ class _CreateModelGroup(GraphQLRequest):
             $sourceColumnId: Int!,
             $labelsetColumnId: Int,
             $name: String!,
-            $modelTrainingOptions: JSONString
+            $modelTrainingOptions: JSONString,
+            $documentModel: Boolean,
         ) {
                 createModelGroup(
                     datasetId: $datasetId,
                     sourceColumnId: $sourceColumnId,
                     labelsetColumnId: $labelsetColumnId,
                     modelTrainingOptions: $modelTrainingOptions,
                     name: $name,
+                    documentModel: $documentModel,
```

Comment:
I would prefer this lives in the modelTrainingOptions, we don't have any other column here to pick a model type or alternatively be a much more generic model enum. Adding a boolean doesn't scale and this really is about picking the model_type.

Reply:
@nkassis : modelTrainingOptions or a modelType enum per your comment on the fog PR?